yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/object_type_mapping.psql

```diff
@@ -17,7 +18,7 @@ JOIN
 LEFT JOIN
     pg_catalog.pg_index pi ON pi.indexrelid = c.oid AND (c.relkind = 'i' OR c.relkind = 'I')
 WHERE 
-    (c.relkind IN ('r', 'p', 'i', 'I', 'm')) AND NOT (c.relkind IN ('i', 'I') AND pi.indisprimary)
+    (c.relkind IN ('r', 'p', 'i', 'I', 'm')) AND NOT (c.relkind ='I' AND pi.indisprimary) -- ignoring partitioned primary keys(no use)
```

Comment:
Why is it okay  to ignore partitioned pk specifically? 
Can you also add context to the PR description? I seem to have forgotten the motivation for making this change of having PK information.

Reply:
We only deal with parent node(of a parent) during assessment, and recommed for it.
So gathering PK info for child nodes(of a partitions) is of no use?

Reply:
Got it, makes sense! pls add that to the comment in code

Reply:
regarding the 3k+ lines changed, yes that is because of PK info being present in TableIndexStats also now, due to which we have json for each PK in the expected assessment report.
pg/osm and pg/omnibus are big tests having lot of tables/PKs